xbmc/Application.h

```diff
@@ -154,6 +154,7 @@ class CApplication : public CXBApplicationEx, public IPlayerCallback, public IMs
 
   bool IsCurrentThread() const;
   void Stop(int exitCode);
+  bool ResumePlayer(const CFileItemPtr & itemIn);
```

Comment:
minor, but same here `ResumePlayback`

Reply:
I don't like Playback, I would like to have the old state back instead of forcing playback.
But this isn't needed anyway. I will create a restorePlayer function inside of powermanagement to get access to FileItem, Stack, Playtime, Playspeed

Reply:
Okay